composer.json

```diff
@@ -47,12 +47,13 @@
         }
     },
     "scripts": {
-        "phpcs": [
-            "vendor/bin/php-cs-fixer fix --verbose --diff --dry-run"
-        ],
-        "tests": [
-            "vendor/bin/phpunit --verbose"
-        ]
+        "test": "vendor/bin/phpunit --stop-on-failure",
```

Comment:
I would prefer not to stop. The testsuite does not take too long, and seeing all failing tests locally might be more beneficial.